lib/whitehall/admin_link_lookup.rb

```diff
@@ -0,0 +1,24 @@
+module Whitehall
+  class AdminLinkLookup
+    include Govspeak::EmbeddedContentPatterns
+
+    def self.find_edition(admin_url)
+      case admin_url
+      when ADMIN_ORGANISATION_CIP_PATH
+        organisation = Organisation.find_by(slug: $1)
```

Comment:
Using these `$` variables gave me pause as it took me a while to realise that `ADMIN_ORGANISATION_CIP_PATH` is probably a regex with a group in it and `$1` and `$2` will be extracting the slugs.

I wonder if changing them to use named groups (`/(?<name>...)` style) would be clearer (except you have to get them from `$~['name']` which is, probably, even more incomprehensible.  Maybe swapping the case for an `if` with explicit `matchdata = regexp.match(admin_url)` would be clearer.

It's a bigger change than we might want to tackle here though, and I'm not sure how much clearer it'd be.

Reply:
That's a good point. I added the keyword parameters in an attempt to make it clearer, but it's still not obvious that it's performing a regex match.

I'm going to leave this for now. We can revisit it if we think of a way to fix this in a readable way.